system-tests/services/test-pods.js

```diff
@@ -1887,7 +1942,14 @@ describe("Services", function() {
                 }
               }
             }
-          ]
+          ],
+          "volumes": [],
+          "fetch": [],
+          "scheduling": {
+            "placement": {
+              "constraints": []
+            }
+          }
```

Comment:
Please expand the piece of code it belongs to. In this particular place we actually need quotes.